lib/Gedmo/Tree/Strategy/ORM/Closure.php

```diff
@@ -254,6 +254,8 @@ public function processPostPersist($em, $entity, AdapterInterface $ea)
             $descendantColumnName = $this->getJoinColumnFieldName($em->getClassMetadata($config['closure'])->getAssociationMapping('descendant'));
             $depthColumnName = $em->getClassMetadata($config['closure'])->getColumnName('depth');
 
+            $referenceId = $em->getClassMetadata($config['closure'])->getAssociationMapping('ancestor')['sourceToTargetKeyColumns'][$ancestorColumnName];
```

Comment:
the min PHP version is 5.3 in the composer.json, and this syntax is not compatible with it